google/resource_container_cluster_test.go

```diff
@@ -635,6 +687,7 @@ resource "google_container_cluster" "with_master_auth" {
 	master_auth {
 		username = "mr.yoda"
 		password = "adoy.rm"
+		issue_client_certificate = false
```

Comment:
Does this change anything from what was there before? I think if you want to test `issue_client_certificate` it probably makes more sense to set it to `true`

Reply:
True is the default behavior, so setting it to false does change, however I think the larger concern is that both cases were not being tested.  I added test specifically to test true/false